app/scripts/HiGlassComponent.js

```diff
@@ -130,12 +130,12 @@ class HiGlassComponent extends React.Component {
     // between views
     this.zoomLocks = {};
     this.locationLocks = {};
+    this.valueScaleLocks = {};
 
     this.prevAuthToken = props.options.authToken;
 
     // locks that keep the value scales synchronized between
     // *tracks* (which can be in different views)
-    this.valueScaleLocks = {};
```

Comment:
Should the comment be moved/removed as well?

Reply:
Yup. Moved.